build.gradle

```diff
@@ -99,9 +101,17 @@ repositories {
 
 dependencies {
     minecraft "net.minecraftforge:forge:${minecraft_version}-${forge_version}"
-    
-    implementation fg.deobf("com.simibubi.create:Create:mc${create_version}")
-    implementation fg.deobf("com.jozufozu.flywheel:Flywheel-Forge:${flywheel_version}")
+
+    jarJar(group: 'com.tterrag.registrate', name: 'Registrate', version: '[MC1.18.2-1.1.3,)') {
+        jarJar.pin(it, project.registrate_version)
+    }
+
+    implementation fg.deobf("com.tterrag.registrate:Registrate:${registrate_version}")

```

Comment:
change this from implementation to compileOnly, helped me to runClient and runData on my end see PR 37

Reply:
Thanks for the heads up.